mage/how-to-guides/run-mage-modules-on-subgraphs.md

```diff
@@ -0,0 +1,139 @@
+---
+id: run-a-subgraph-module
+title: How to run a MAGE module on subgraphs
+sidebar_label: Run modules on subgraphs
+---
+
+If you have already read our how-to guide on [how to run MAGE modules and graph algorithms](/mage/how-to-guides/run-a-query-module.md),
+now it's time to move on to the finer execution of graph analytics.
+In this how-to guide, we will project a portion of the graph from the whole network persisted in 
+Memgraph, and run our algorithms on that portion of the graph.
+
+### Why might running an algorithm across the entire network not be ideal for you?
+
+If you didn't observe already, by executing a query module (or a MAGE module, which is essentially
+the same thing), the algorithm is executed on the whole network. This is impractical for multiple
+reasons:
+- your graph may be heterogeneous, and you want the module to run only on specific labels.
+- your graph may be too large, and you only want a portion of it to be updated with the analytics
+- you have multiple diverse data models and graphs in your network, and
+running analytics on all the graphs at the same time just does not make sense
+
+For that purpose, Memgraph has enabled you to execute modules on subgraphs and graph
+projections, to make the insights yielded by graph algorithms affect only the
+necessary nodes in your graph, saving you time and making the data more consistent
+and up to its specifications.
```

Comment:
```suggestion
### When not to run algorithms across the entire network and use the projection feature?

Executing any MAGE query module, the algorithm is executed on the whole network. 
This is impractical in the following use cases: 
- if the graph is heterogeneous, and you want to run the module only on specific labels
- if the graph is too large, and you only want to use the analytics to update only a portion of it 
- the network contains multiple diverse data models and graphs, and running analytics on all the graphs at the same time IS WHAT? 

That is why Memgraph enables module execution on subgraphs and graph
projections. The insights yielded by graph algorithms can then affect only the necessary nodes in your graph, thus saving time WITH WHAT, making the data more consistent and up to its specifications. 
```

the network contains multiple diverse data models and graphs, and running analytics on all the graphs at the same time IS WHAT? 

The insights yielded by graph algorithms can then affect only the necessary nodes in your graph, thus saving time WITH WHAT, making the data more consistent and up to its specifications.

Reply:
Will do.